### PR TITLE
Add State Of Rust triage procedure

### PR DESCRIPTION
**Pull request by Aaronepower**
[Rendered](https://github.com/Aaronepower/rust-forge/blob/master/triage-procedure.md#state-of-rust-triage)

cc @Mark-Simulacrum

### CODE REVIEW

**Comment by BatmanAoD** [comment]
A few things to resolve:

 * Sometimes the active RFC is superseded without being closed; this is not always obvious without clicking through to the RFC, then to the tracking issue, then to links in the tracking issue, finally arriving at a *new* RFC and associated tracking issue.
     * It's probably worth explicitly mentioning in the procedure that the linked RFC may have been superseded.
     * Unfortunately, I'm not sure it's always clear which RFC is "active"...when there's confusion, should we just ask for feedback in the related RFC threads?
 * Should we have guidelines on what conditions warrant pinging the participants in the tracking-issue thread? For instance, if a tracking issue hasn't been updating in a few weeks, but you suspect (for some reason) that work may have been done without updating the issue, does that warrant a ping?

**Comment by Aaronepower** [comment]
Yes, I meant to have an item about pinging, I just forgot it. I was thinking if there's been seemingly no activity for 21 days/3 weeks. I was wondering who should be pinged though? The creator of the tracking issue? The team responsible for the RFC?

**Comment by BatmanAoD** [comment]
@Aaronepower I think it probably depends on the state of the issue. From most to least specific:

 * If someone has started implementing it: ping them; if they've been pinged already and haven't responded for a week (or two?), consider it an "orphaned" implementation
 * If no one has started implementing it, or if it's "orphaned": ping the relevant team for *implementing* (probably the compiler team for language features?)
 * If there's no clear choice for the team that should be doing the implementation, ping the team and/or individual(s) responsible for the RFC

**Comment by Aaronepower** [comment]
@BatmanAoD I've added in those instructions, feel free to review!

**Comment by BatmanAoD** [comment]
Looks good! I've submitted some minor edits as a PR against your fork's master branch.

File: triage-procedure.md [hunk 1]
```diff
@@ -234,3 +234,37 @@ and add tags as you see fit. The following categories should, ideally, be assign
...
+     * If there has been a decision to deprecate the RFC, move that to the
+        “Deprecated” column.
+6. If there have been no meaningful changes to the RFC within 21 days,
+   ping someone for an update on the status of the feature.
+     * If there have been PRs implementing the RFC, ping the author(s).
+     * If author has not responded within a week, or there are no relevant PRs,
+       ping the relevant team.
+     * If there is no clear choice for the team that should be doing the
+       implementation, ping the team and/or individual(s) responsible for
+       the RFC.
```
Comment (Mark-Simulacrum) [inline triage-procedure.md]: FWIW, I think this should essentially never happen. Could we add a "please add this to release team meeting notes" so we can discuss what went wrong to let this happen? Ideally all RFCs are pretty much well placed to be implemented by a concrete team(s).

**Comment by Mark-Simulacrum** [comment]
Okay, let's merge this! Thanks!

I think it might be worth talking to @Centril about this as well to possibly help integrate this into the normal RFC merge procedure, since they've been doing quite a bit of that lately :)

File: triage-procedure.md (hunk 1 above)
Comment (Centril) [reply triage-procedure.md]: Yeah I cannot recall an instance where I made a tracking issue with no responsible team.

**Pull request merged**